nautobot_device_lifecycle_mgmt/tables.py

```diff
@@ -220,12 +220,31 @@ class Meta(BaseTable.Meta):  # pylint: disable=too-few-public-methods
 class DeviceSoftwareValidationResultTable(BaseTable):
     """Table for device software validation report."""
 
-    name = tables.Column(accessor="device__device_type__model", verbose_name="Device Type")
-    total = tables.Column(accessor="total", verbose_name="Total")
-    valid = tables.Column(accessor="valid", verbose_name="Valid")
-    invalid = tables.Column(accessor="invalid", verbose_name="Invalid")
-    no_software = tables.Column(accessor="no_software", verbose_name="No Software")
+    name = tables.TemplateColumn(
+        template_code='<a href="/dcim/devices/?device_type_id={{ record.device__device_type__pk }}">{{ record.device__device_type__model }}</a>'
+    )
```

Comment:
What's the intention here? The url takes us to the device list view. Should this take us to the device software validation results?

Reply:
For that one I was thinking of it going to the device list that has that type of hardware and the "Total" Column sending the user to the device software validation results of that device type. I was trying to make it to where if the user wanted a quick way to see all their devices to device type they could.

Let me know if you think we should make them go to the validation results for both links.